### ISSUE TITLE
error on empty folder ? 

### ISSUE THREAD

**Issue by redcinelli** [athityakumar/colorls#41]
Hey there, first off awesome project.
I just installed it and it look really neat !

But I happend to use it against some empty directory and I got that 

```bash
uncoolshogun@tpx1:~$ mkdir test
uncoolshogun@tpx1:~$ sl test/
/var/lib/gems/2.3.0/gems/colorls-0.1.4/lib/colorls/core.rb:103:in 'in_line': undefined method '+' for nil:NilClass (NoMethodError)
        from /var/lib/gems/2.3.0/gems/colorls-0.1.4/lib/colorls/core.rb:87:in 'chunkify'
        from /var/lib/gems/2.3.0/gems/colorls-0.1.4/lib/colorls/core.rb:20:in 'ls'
        from /var/lib/gems/2.3.0/gems/colorls-0.1.4/lib/colorls/flags.rb:23:in 'block in process'
        from /var/lib/gems/2.3.0/gems/colorls-0.1.4/lib/colorls/flags.rb:21:in 'each'
        from /var/lib/gems/2.3.0/gems/colorls-0.1.4/lib/colorls/flags.rb:21:in 'process'
        from /var/lib/gems/2.3.0/gems/colorls-0.1.4/exe/colorls:4:in '<top (required)>'
        from /usr/local/bin/colorls:23:in 'load'
        from /usr/local/bin/colorls:23:in '<main>'
uncoolshogun@tpx1:~$ 

```

I am not familiar with ruby but it does look like a stack trace :stuck_out_tongue_closed_eyes: 
Is it just me ? I am on ubuntu 16.04 

ps : sl is my alias for colorls ^-^

**Comment by redcinelli** [comment]
Oy ! Buddy 

I might I have found a fix :v: 

```ruby
  def ls
      unless @contents.empty?
        @contents = chunkify
        @max_widths = @contents.transpose.map { |c| c.map(&:length).max }
        @contents.each { |chunk| ls_line(chunk) }
        print "\n"
        display_report if @report
      else
        print "it is empty baby \uf119\n".colorize(:blue)
      end

      true
  end
```

I did fork your repo and red the contributing handbook but i've never done a pull request before, how are your/the best practice.

cheers :tada:

**Comment by soctar** [comment]
In a similar vein, lc on a single file also fails - might be the same fix.

**Comment by athityakumar** [comment]
@redcinelli @soctar - Thanks for filing this issue. I've reviewed PR #46 regarding this issue. 😄

**Comment by lzjqsdd** [comment]
same,hope to fix it

**Comment by athityakumar** [comment]
Closed by PR #46
